design/design-doc-stacks.md

```diff
@@ -307,18 +393,21 @@ Possible solutions for package processing are listed below, along with their pro
   * PRO: This is the most versatile option as it can be used in contexts outside of a Crossplane cluster.
   * CON: Doesn't integrate very cleanly into the Crossplane mainline scenario with the SM.
 
-**Recommendation:** It is recommended to start with the "Job initContainer and shared volume" approach that copies the package contents from the package initContainer to a shared volume and has a Crossplane container main entry point perform the processing logic over the shared volume contents, sending out the artifacts to `stdout`.
-Stack packages should only have a few KB of yaml files, so it's a significant concern to blow that image size up to many MB right now.
+One approach could be to start with the "Job initContainer and shared volume" that copies the package contents from the package initContainer to a shared volume and has a Crossplane container main entry point perform the processing logic over the shared volume contents, sending out the artifacts to `stdout`.
 The processing/unpacking logic can easily move to its own image that can be used as a base layer in the future (or a CLI tool), so this approach is not very divergent.
-It is a good place to start and iterate over as we learn more without investing much effort that cannot be reused.
+
+Each of these designs offer a good place to start.  Through iteratation over time we will learn more, hopefully without investing much effort that cannot be reused.
 
 ## Questions and Open Issues
 
 * Dependency resolution design: [#434](https://github.com/crossplaneio/crossplane/issues/434)
-* Updating/Upgrading Extension: [#435](https://github.com/crossplaneio/crossplane/issues/435)
+* Updating/Upgrading Stacks: [#435](https://github.com/crossplaneio/crossplane/issues/435)
```

Comment:
we updated Extension -> Stack here, but not in the rest of the list items.  Should we make that naming update more broadly in this list?

Reply:
This PR chases #573 which started the renaming.  IIRC, this line was on a merge conflict boundary.  Since the issue is titled "Extensions versioning and upgrading design" I probably should have stuck with Extensions here.

I'll look for other occurrences of Extensions in this PR and roll the renaming dice.

Reply:
Revised in 8c72667f0da6d4f47ea5b9f4f0d49b773e011c9e